.github/ISSUE_TEMPLATE/depr-ticket.yml

```diff
@@ -0,0 +1,84 @@
+name: Deprecation (DEPR) Ticket
```

Comment:
Many of the other PRs are missing this template, why does this one have it?

Reply:
Those repos have Issues turned on already. Since this repo has issues turned off, I committed a config.yml file to configure the Issues such that only the DEPR template is shown. 

The DEPR ticket template here is defined in .github repo, however, once you specify a custom Issue config, you cannot inherit ticket templates from .github. So, I have to copy this template here.

Once we can open up Issues of all types, I'd expect to throw this and the config file away and inherit the org default.

Reply:
Cool, adding something like this as a comment in this template might be useful for future context but no worries if it's a pain to add now that the PRs are up.

Reply:
It would not be difficult to do, although I'd probably just want to rerun my scripts and make follow up PRs, since this has been merged to many repos already.